client/state/themes/actions.js

```diff
@@ -651,11 +658,12 @@ export function deleteTheme( themeId, siteId ) {
 			siteId,
 		} );
 		return wpcom.undocumented().deleteThemeFromJetpack( siteId, themeId )
-			.then( () => {
+			.then( ( theme ) => {
```

Comment:
You could destructure and rename the arg here in one go,

```js
.then( ({ name: themeName } ) => {
```

Not that it matters 😁